### PR TITLE
add more attributes to marketcap api

### PR DESCRIPTION
**Pull request by shadowcrypto1**

### CODE REVIEW

File: config-sample.py [hunk 1]
```diff
@@ -10,6 +10,10 @@
 REDIS_MARKETCAP_DB = 0
 REDIS_PRICE_DB = 1
 update_timeout = 60
+TOTAL_SUPPLY_ALL_CHAINS_REDIS_PREFIX = 'TOTAL_SUPPLY_ALL_CHAINS_'
```
Comment (MiKoronjoo) [inline config-sample.py]: From my understanding, the variable represents the chain total supply, so naming it `CHAIN_TOTAL_SUPPLY` would be more appropriate than `TOTAL_SUPPLY_ALL_CHAINS`.

File: config-sample.py [hunk 2]
```diff
@@ -10,6 +10,10 @@
...
+SUPPLY_IN_BRIDGE_CONTRACTS_REDIS_PREFIX = 'SUPPLY_IN_BRIDGE_CONTRACTS_'
+SUPPLY_IN_VEDEUS_CONTRACT_REDIS_PREFIX = 'SUPPLY_IN_VEDEUS_CONTRACT_'
+TOTAL_SUPPLY_REDIS_PREFIX = 'TOTAL_SUPPLY_'
```
Comment (MiKoronjoo) [inline config-sample.py]: This line is duplicated from line 18.

File: server.py [hunk 3]
```diff
@@ -82,10 +102,13 @@ def get_deus_info(route):
         return jsonify(status='error', msg='Route not found'), 404
     total_supply = 0
     circulating_supply = 0
+    non_circulating_supply = 0
     price = float(marketcap_db.get(PRICE_REDIS_TAG))
     for chain in deus_chains:
         supply = int(marketcap_db.get(TOTAL_SUPPLY_REDIS_PREFIX + chain))
-        circulating_supply += supply - int(marketcap_db.get(NC_SUPPLY_REDIS_PREFIX + chain))
+        ncSupply = int(marketcap_db.get(NC_SUPPLY_REDIS_PREFIX + chain))
```
Comment (MiKoronjoo) [inline server.py]: A variable name in Python should be snake_case.

File: updater.py [hunk 4]
```diff
@@ -48,6 +52,7 @@ def deus_updator(managers):
         else:
             print('TOTAL SUPPLY:   ', total_supply)
             print('NON-CIRCULATING:', nc_supply)
+            print('CIRCULATING:', total_supply- nc_supply)
```
Comment (MiKoronjoo) [inline updater.py]: According to PEP 8 guidelines, it is recommended to include a single space on both sides of binary operators.

File: updater.py [hunk 5]
```diff
@@ -86,6 +91,7 @@ def xdeus_updator(managers):
...
```
Comment (MiKoronjoo) [inline updater.py]: We have previous issue here as well

**Review by MiKoronjoo — CHANGES_REQUESTED** [review]

File: config-sample.py (hunk 1 above)
Comment (shadowcrypto1) [reply config-sample.py]: Modified the variable name

File: config-sample.py (hunk 2 above)
Comment (shadowcrypto1) [reply config-sample.py]: Removed the duplicated constant

File: server.py (hunk 3 above)
Comment (shadowcrypto1) [reply server.py]: changed the variable name to follow snake case

File: updater.py (hunk 4 above)
Comment (shadowcrypto1) [reply updater.py]: Addressed the issue accordingly

File: updater.py (hunk 5 above)
Comment (shadowcrypto1) [reply updater.py]: Addressed the issue accordingly

**Review by MiKoronjoo — COMMENTED** [review]
All good

**Pull request merged**